src/server/server.ts

```diff
@@ -53,12 +55,12 @@ const startAppServer = (
     app.use(express.urlencoded({ extended: true }))
     app.use(cookieParser())
     app.use(express.static(path.join(__dirname, '../public')))
-    app.use('/admin', adminRouter)
-    app.use('/api', setupAuth(adminPass), apiRouter)
+    app.use('/admin', setupAuth, adminRouter)
+    app.use('/api', setupAuth, apiRouter)
```

Comment:
why not run `setupAuth` for all requests? Cuz it doesn't reject, only modifies req object.

```
app.use(setupAuth)
app.use('/admin', adminRouter)
...
```